### ISSUE TITLE
Document public meta-data

### ISSUE THREAD

**Issue by k-rus** [Starcounter/Starcounter#2058]
The documentation should explain the structure of the public meta-data and provide information how to use them from application code and interactive SQL in Administrator.

This issue depends on #2028.

**Comment by k-rus** [comment]
Some documentation was published on stackoverflow in [this answer](http://stackoverflow.com/a/39490683/840340)

**Comment by k-rus** [comment]
Closing in favor of newer issue https://github.com/Starcounter/Starcounter/issues/3001
